datumaro/plugins/open_images_format.py

```diff
@@ -15,29 +16,32 @@
 import types
 
 from attr import attrs
+import cv2
+import numpy as np
 
+from datumaro.cli.util import make_file_name
```

Comment:
I don't think importing from `cli` is good here. Probably, this function is better be moved to the core utils (and it is done in the versioning PR).

Reply:
Yeah, fair enough. I'll probably steal your move patch (unless you want to submit it as a separate PR).

Reply:
Patch stolen. 🙂